router/throttler/factory_test.go

```diff
@@ -52,6 +52,25 @@ func TestFactory(t *testing.T) {
 			}, 2*time.Second, 100*time.Millisecond, "limit: %d, expectedLimit: %d", ta.getLimit(), maxLimit*5/10)
 		})
 	})
+
+	t.Run("when throttlerV2 is false", func(t *testing.T) {
+		config := config.New()
```

Comment:
[optional]: Variable 'config' collides with imported package name 
```suggestion
		conf := config.New()
```